### DISCUSSION TITLE
I recommend adding a new feature

### DISCUSSION THREAD

**Discussion by Get70** [rxhanson/RectanglePro-Community#183] [Q&A]
Can I recommend adding a new feature?
I hope users can have a window always on top of other windows by clicking or choosing one application or window, which I think is a very important requirement to users. It's usually called a pin window.
Thanks!

**Comment by rxhanson** [comment]
Unfortunately, macOS doesn't have a supported way to do this. My application can make its own windows stay on top (i.e. the Preferences window), but it cannot make windows belonging to other applications stay on top. From what I can tell, there used to be other apps that would do this years ago, which were broken by macOS, and I think they might have built workarounds that required users to disable SIP. As such, this isn't something I would consider adding. Sorry!

**Reply by ssbarnea** [reply]
Maybe a workaround for this would be to auto resize any other active window to avoid it overlapping to the supposedly always on top one?

The main idea is that I do not wan to lose the video meeting window behind other ones.
